run-package-command.sh

```diff
@@ -0,0 +1,3 @@
+#/usr/bin/env bash
+
+lerna exec --scope $1 npm run $2 `[[ -n "${@:3}" ]] && echo "-- -- ${@:3}"`
```

Comment:
Isn't it supposed to be:
```suggestion
lerna exec --scope $1 -- npm run $2 `[[ -n "${@:3}" ]] && echo "-- ${@:3}"`
```
?

Reply:
yeah that works too, not sure if there's difference. maybe easier to understand as opposed to double `-- --`?